lib/pages/current_available_meetings_page.dart

```diff
@@ -0,0 +1,106 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:vot_senat_client/bloc/available-meetings-bloc/available-meetings-bloc.dart';
+import 'package:vot_senat_client/bloc/available-meetings-bloc/available-meetings-event.dart';
+import 'package:vot_senat_client/bloc/available-meetings-bloc/available-meetings-state.dart';
+import 'package:vot_senat_client/model/meeting.dart';
+
+class AvailableMeetingsPage extends StatefulWidget {
+  const AvailableMeetingsPage({Key? key}) : super(key: key);
+
+  @override
+  State<AvailableMeetingsPage> createState() => _AvailableMeetingsState();
+}
+
+class _AvailableMeetingsState extends State<AvailableMeetingsPage> {
+  List<Meeting> cards = [
+    new Meeting(id: 1, title: 'Sedinta 1', date: '05/07/2021'),
+    new Meeting(id: 2, title: 'Sedinta 2', date: '08/07/2021'),
+    new Meeting(id: 3, title: 'Sedinta 3', date: '12/09/2021'),
+  ];
+
+  @override
+  void initState() {
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Available meetings page"),
+        centerTitle: true,
+      ),
+      body: ListView.builder(
+        itemCount: cards.length,
+        itemBuilder: (context, index) {
+          return Padding(
+            padding: const EdgeInsets.symmetric(vertical: 10, horizontal: 5),
+            child: PhysicalModel(
```

Comment:
I would suggest adding the component under PhysicalModel in a new widget called, meeting card